geco/mips/miplib/loader.py

```diff
@@ -0,0 +1,71 @@
+import tempfile
+import requests
+import pyscipopt as scip
+import os
+import pandas as pd
+
+INSTANCES_DIR = tempfile.gettempdir() + "/geco/miplib/instances/"
+MIPLIB_INSTANCE_URL = "https://miplib.zib.de/WebData/instances/"
+
+
+def load_instances(filters={}, instances_csv=None):
```

Comment:
While we have not general guide for this, I think we should avoid mutable default arguments and use 'None' instead.

Reply:
agreed. done.